repository/flink/docs/demo/tests/flink-demo/01-update-storage-class.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl delete storageclass standard
+  - command: kubectl apply -f https://raw.githubusercontent.com/rancher/local-path-provisioner/master/deploy/local-path-storage.yaml
```

Comment:
This is applying the same `StorageClass` that has just been deleted, see https://github.com/rancher/local-path-provisioner/blob/master/deploy/local-path-storage.yaml#L82. Why is this necessary to do here? This also means that the error condition above will always fire.

Reply:
@kensipe I see this as an infra step. If we run these tests along with other tests present in this repository we might be changing the outcome of the tests. What about just using the storage class that is already present and which is being deleted?

Reply:
yeah... these tests are great for setting up and running / verifying the flink demo... perhaps we shouldn't hook them into the infra... is there any suggestion?